docs/clp-adr.md

```diff
@@ -0,0 +1,206 @@
+# ADR 001: CLP integration with Sifnode
+
+## Changelog
+
+- 2020/11/02 Initial version
+
+## Status
+
+*Proposed*
+
+## Context
+This ADR contains a summary of decisions taken during the implementation of the clp module and a summary of the final outcome
+### Summary
+
+For the Sifchain MVP ,CLP module provides the following functionalities 
+- Create New Liquidity Pool
+- Add Liquidity to an Existing Liquidity pool 
+- Remove Liquidity from an Existing Liquidity pool 
+- Swap tokens  
+        -Swap an External token for Native or vice versa (single swap)    
+        -Swap an External token for another External Token (double swap) 
+- Decommission an Existing Liquidity pool 
+
+### Basic Terminology 
+-Asset : An asset is most basic unit of a CLP . It Contains source chain, symbol and ticker to identify a token .
+```golang
+SourceChain: ETHEREUM
+Symbol: ETH
+Ticker: ceth
+
+SourceChain: SIFCHAIN
+Symbol: RWN
+Ticker: rwn
```

Comment:
need to update this too `rowan` to match smart-contact land.

Reply:
both the ticker and the symbol ?

Reply:
@Jedi2002 please open up an issue for us to disucss the ticker vs symbol conversation and please set the context for us as this is a little unclear for me right now. Then we can close the loop on this ADR and update the code as needed too. Thanks